ledger/participant-integration-api/BUILD.bazel

```diff
@@ -295,11 +223,73 @@ da_scala_test_suite(
         "-Djava.security.debug=\"certpath ocsp\"",  # This facilitates debugging of the OCSP checks mechanism
     ],
     resources = glob(["src/test/resources/**/*"]),
-    scala_deps = scala_test_deps,
-    deps = test_deps,
+    scala_deps = [
+        "@maven//:com_typesafe_akka_akka_actor",
+        "@maven//:com_typesafe_akka_akka_stream",
+        "@maven//:com_typesafe_akka_akka_stream_testkit",
+        "@maven//:org_mockito_mockito_scala",
+        "@maven//:org_scalacheck_scalacheck",
+        "@maven//:org_scalactic_scalactic",
+        "@maven//:org_scalatest_scalatest",
+        "@maven//:org_scalatestplus_scalacheck_1_14",
+        "@maven//:org_scalaz_scalaz_core",
+    ],
+    deps = [
+        ":participant-integration-api",
+        ":participant-integration-api-tests-lib",
+        "//bazel_tools/runfiles:scala_runfiles",
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/archive:daml_lf_dev_archive_proto_java",
+        "//daml-lf/data",
+        "//daml-lf/engine",
+        "//daml-lf/language",
+        "//daml-lf/transaction",
+        "//daml-lf/transaction:value_proto_java",
+        "//daml-lf/transaction-test-lib",
+        "//language-support/scala/bindings",
+        "//ledger-api/rs-grpc-akka",
+        "//ledger-api/rs-grpc-akka:rs-grpc-akka-tests-lib",
+        "//ledger-api/rs-grpc-bridge",
+        "//ledger-api/sample-service",
+        "//ledger-api/testing-utils",
+        "//ledger/caching",
+        "//ledger/ledger-api-client",
+        "//ledger/ledger-api-common",
+        "//ledger/ledger-api-common:ledger-api-common-scala-tests-lib",
+        "//ledger/ledger-api-domain",
+        "//ledger/ledger-api-health",
+        "//ledger/ledger-resources",
+        "//ledger/ledger-resources:ledger-resources-test-lib",
+        "//ledger/metrics",
+        "//ledger/participant-state",
+        "//ledger/participant-state-index",
+        "//ledger/test-common",
+        "//ledger/test-common:dar-files-stable-lib",
+        "//libs-scala/concurrent",
+        "//libs-scala/contextualized-logging",
+        "//libs-scala/grpc-utils",
+        "//libs-scala/ports",
+        "//libs-scala/postgresql-testing",
+        "//libs-scala/resources",
+        "//libs-scala/resources-akka",
+        "//libs-scala/resources-grpc",
+        "@maven//:ch_qos_logback_logback_classic",
+        "@maven//:com_github_ben_manes_caffeine_caffeine",
+        "@maven//:com_zaxxer_HikariCP",
+        "@maven//:commons_io_commons_io",
+        "@maven//:io_dropwizard_metrics_metrics_core",
+        "@maven//:io_grpc_grpc_netty",
+        "@maven//:io_netty_netty_handler",
+        "@maven//:io_netty_netty_transport",
+        "@maven//:org_flywaydb_flyway_core",
+        "@maven//:org_mockito_mockito_core",
+        "@maven//:org_reactivestreams_reactive_streams",
+        "@maven//:org_slf4j_slf4j_api",
+    ],
 )
 
-da_scala_test_suite(
+# The "manual" test tag doesn't seem to work with `da_scala_test_suite`.
```

Comment:
I’m confused. You switched from `da_scala_test_suite` but then you’re still not using `manual`. Can we do one or the other?

Reply:
Yeah, that was my original plan and then I switched it. I'll put it back.

Reply:
I’d prefer the manual option and fixing `da_scala_test_suite`.

Reply:
I did try to fix `da_scala_test_suite` but I can't find the problem. Can you see one?

I think it might be an issue in `scala_test_suite`, but I can't be sure.

Reply:
Switched to manual execution with the tag.

Reply:
hm, I thought it might not pass the tag along but doesn’t seem to be it. Fine to leave it out of this PR.

Reply:
Looking at the [implementation of `scala_test_suite`](https://github.com/bazelbuild/rules_scala/blob/b85d1225d0ddc9c376963eb0be86d9d546f25a4a/scala/private/rules/scala_test.bzl#L125-L144), it does forward `kwargs` to the individual `scala_test` targets, but it doesn't forward any tags to the final `test_suite` target. So, the final `test_suite` target will still be non-manual. I'd say that's a bug in rules_scala.